src/components/search/SearchBar.vue

```diff
@@ -100,6 +100,9 @@ export default class SearchBar extends Vue {
   }
 
   async autocomplete(q: string): Promise<void> {
+    if (q.length < 2) {
```

Comment:
Might want to also add a hint that says to type more than 2 characters, but I can add that to an issue